Exercise_1.py

```diff
@@ -1,18 +1,29 @@
 class myStack:
      def __init__(self):
```

Comment:
You should use an array and implement stack. You are not recommended to use inbuilts like append() and pop()